hack/generate_override_snapshot_yaml.sh

```
@@ -0,0 +1,222 @@
+    done
+
+    if [ ${#missing[@]} -gt 0 ]; then
+        error "Missing required tools: ${missing[*]}"
+        exit 1
+    fi
+}
+
+get_image_sha(){
+	local image_ref="${1}"
+	image_ref=${image_ref#*@} # Drop anything before the 'sha:'
+	echo ${image_ref%\"} # Remove the trailing quote
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)